src/main/java/org/prebid/server/settings/model/Account.java

```diff
@@ -15,6 +15,8 @@
 
     Integer videoCacheTtl;
 
+    Integer analyticsSamplingFactor;
```

Comment:
Pls declare this field after `enforceGdpr` flag. Check all other occurrences to be in same style.